aiohttp_pydantic/oas/__init__.py

```diff
@@ -13,14 +13,16 @@ def setup(
     apps_to_expose: Iterable[web.Application] = (),
     url_prefix: str = "/oas",
     enable: bool = True,
+    version_spec: Optional[str] = None,
+    title_spec: Optional[str] = None,
 ):
     if enable:
         oas_app = web.Application()
         oas_app["apps to expose"] = tuple(apps_to_expose) or (app,)
         oas_app["index template"] = jinja2.Template(
             resources.read_text("aiohttp_pydantic.oas", "index.j2")
         )
-        oas_app.router.add_get("/spec", get_oas, name="spec")
+        oas_app.router.add_get("/spec", get_oas(version_spec, title_spec), name="spec")
```

Comment:
Could you pass parameter `version_spec` and `title_spec` using `oas_app["version spec"]` and `oas_app["title spec"]`